/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 27 1 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.551560163498


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_27_1_50_4_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 26 1 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.486738920212


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_26_1_50_9_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 24 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.491743087769


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_24_2_50_9_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 22 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.491609096527


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_22_2_50_16_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 20 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.498347043991


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_20_2_50_25_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 18 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.488752126694


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_18_2_50_36_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 16 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.491953134537


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_16_2_50_49_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 14 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.489624023438


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_14_2_50_64_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 12 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.48776102066


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_12_2_50_81_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 10 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.487515926361


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_10_2_50_100_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test none no_weight_dependence postpre 8 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.501113891602


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_none_8_2_50_121_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
